READMEs/marketplace-flow.md

```diff
@@ -276,19 +276,11 @@ print(f"order_tx_id = '{order_tx_id}'")
 
 # Bob downloads. If the connection breaks, Bob can request again by showing order_tx_id.
 file_path = ocean.assets.download_asset(
-    asset,
-    service.service_endpoint,
-    bob_wallet,
-    './',
-    order_tx_id
-)
-
-file_path = ocean.assets.download(
-    asset.did,
-    service.index,
-    bob_wallet,
-    order_tx_id,
-    destination='./'
+    asset=asset,
+    provider_uri=config.provider_url,
```

Comment:
again, why do I need the provider_uri here? I have an access service in the asset. That access service has a service_endpoint. No need for provider_uri.